src/redux/reducers/meta/markets.js

```diff
@@ -85,6 +85,21 @@ export default (state = getInitialState(), action = {}) => {
       }, EMPTY_OBJ)
       return newState
     }
+    case marketTypes.SET_CCYS_ID: {
+      const { ids: [idsArray] } = payload
+      const marketsKeysArray = Object.keys(state)
+
+      const newState = _reduce(marketsKeysArray, (acc, key) => {
+        const market = state[key]
+        const idArray = _find(idsArray, (element) => element[0] === market.base, null)
+        if (!idArray || !idArray[1]) {
+          return acc
+        }
+        acc[key] = { ...market, baseCCYid: idArray[1] }
+        return acc
+      }, EMPTY_OBJ)
+      return newState
+    }
```

Comment:
I dont see advantage or good reason for storing article-id in `markets` state and unnecessarily polluting markets state with extra data, we should store in separate state like state.ccyZendeskIds and save it as object in form of { [base]: [id] }, so its easily retrievable using activeMarket.base